main_test.go

```diff
@@ -18,16 +18,15 @@ var dbsetup = false
 
 func TestMain(m *testing.M) {
 
-	// viper.Set("DbConnection", os.Getenv("TEST_DATABASE_URL"))
-	viper.Set("DbConnection", os.Getenv("dbname=ts"))
+	viper.Set("DbConnection", os.Getenv("TEST_DATABASE_URL"))
```

Comment:
This looks like a thing where you uncomment / comment lines to debug or something, did you mean to check this change in?

Reply:
I've flagged that one for comment in the PR to the upstream repo.  It is needed for the tests to run on my environment, but I'm worried that I may have missed a better way to handle it.